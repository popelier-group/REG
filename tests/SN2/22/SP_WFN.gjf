# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.25732900   -0.00006700    0.00016300
 H                 -0.97493400   -0.88429100   -0.55075100
 H                 -0.97485000    0.91914800   -0.49009400
 H                 -0.97500500   -0.03509700    1.04140000
 Br                 1.64818200    0.00001100   -0.00001800
 Cl                -3.20388400    0.00000500   -0.00003100

frame.wfn
