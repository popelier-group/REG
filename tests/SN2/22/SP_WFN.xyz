
6

C       -0.496753    0.497651    0.895414
H       -0.271946   -0.507116    1.219466
H       -1.231192    0.539304    0.105498
H        0.382706    1.090528    0.695024
Br        0.771187   -0.772271   -1.389672
Cl       -1.346506    1.348285    2.426237
