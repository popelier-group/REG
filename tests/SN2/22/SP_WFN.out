 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.22/Gau-73534.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.22/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     73535.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.25733  -0.00007   0.00016 
 H                    -0.97493  -0.88429  -0.55075 
 H                    -0.97485   0.91915  -0.49009 
 H                    -0.97501  -0.0351    1.0414 
 Br                    1.64818   0.00001  -0.00002 
 Cl                   -3.20388   0.00001  -0.00003 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.257329   -0.000067    0.000163
      2          1           0       -0.974934   -0.884291   -0.550751
      3          1           0       -0.974850    0.919148   -0.490094
      4          1           0       -0.975005   -0.035097    1.041400
      5         35           0        1.648182    0.000011   -0.000018
      6         17           0       -3.203884    0.000005   -0.000031
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.079400   0.000000
     3  H    1.079399   1.804459   0.000000
     4  H    1.079402   1.804460   1.804455   0.000000
     5  Br   2.905511   2.822416   2.822284   2.822569   0.000000
     6  Cl   1.946555   2.460384   2.460402   2.460429   4.852066
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.496753    0.497651    0.895414
      2          1           0       -0.271946   -0.507116    1.219466
      3          1           0       -1.231192    0.539304    0.105498
      4          1           0        0.382706    1.090528    0.695024
      5         35           0        0.771187   -0.772271   -1.389672
      6         17           0       -1.346506    1.348285    2.426237
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         154.0063085           0.8467894           0.8467894
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.2278438309 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  3.08D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.39885842     A.U. after   12 cycles
            NFock= 12  Conv=0.10D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.56240-101.39413 -62.21556 -56.03343 -56.03323
 Alpha  occ. eigenvalues --  -56.03323 -10.08235  -9.30781  -8.42440  -7.07118
 Alpha  occ. eigenvalues --   -7.06331  -7.06331  -6.24781  -6.24695  -6.24695
 Alpha  occ. eigenvalues --   -2.36014  -2.35988  -2.35988  -2.35920  -2.35920
 Alpha  occ. eigenvalues --   -0.68364  -0.54381  -0.49443  -0.30264  -0.30264
 Alpha  occ. eigenvalues --   -0.24925  -0.16581  -0.16581  -0.06029  -0.05969
 Alpha  occ. eigenvalues --   -0.05969
 Alpha virt. eigenvalues --    0.11217   0.12110   0.15100   0.15100   0.17332
 Alpha virt. eigenvalues --    0.17684   0.19522   0.19522   0.20864   0.20865
 Alpha virt. eigenvalues --    0.21524   0.22639   0.22640   0.24827   0.28817
 Alpha virt. eigenvalues --    0.31242   0.31242   0.37192   0.40321   0.53963
 Alpha virt. eigenvalues --    0.53963   0.55632   0.58430   0.59168   0.59168
 Alpha virt. eigenvalues --    0.61911   0.61912   0.67662   0.77011   0.77011
 Alpha virt. eigenvalues --    0.79794   0.83333   0.87222   0.87222   0.88394
 Alpha virt. eigenvalues --    0.88395   0.96185   0.99418   0.99418   1.02641
 Alpha virt. eigenvalues --    1.02641   1.04145   1.14305   1.25684   1.25685
 Alpha virt. eigenvalues --    1.25872   1.56067   1.67060   1.67060   1.85295
 Alpha virt. eigenvalues --    1.86872   1.86872   2.04519   2.20351   2.20355
 Alpha virt. eigenvalues --    2.27341   2.27343   2.32141   2.45254   2.45255
 Alpha virt. eigenvalues --    2.55939   2.62131   2.63954   2.63955   2.69866
 Alpha virt. eigenvalues --    2.69866   3.14973   3.22905   3.86662   3.86663
 Alpha virt. eigenvalues --    4.38978   4.38979   4.39895   4.39897   4.55385
 Alpha virt. eigenvalues --    6.77137   7.79796   7.79801   7.86809   9.83895
 Alpha virt. eigenvalues --   24.04095  25.99206  25.99206  26.46897  48.08314
 Alpha virt. eigenvalues --  215.82376 290.00647 290.00647 290.072181020.84460
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.584442   0.412803   0.412805   0.412808  -0.057645  -0.176301
     2  H    0.412803   0.518327  -0.037913  -0.037921  -0.006348  -0.040720
     3  H    0.412805  -0.037913   0.518311  -0.037920  -0.006352  -0.040712
     4  H    0.412808  -0.037921  -0.037920   0.518335  -0.006334  -0.040728
     5  Br  -0.057645  -0.006348  -0.006352  -0.006334  35.903250   0.056373
     6  Cl  -0.176301  -0.040720  -0.040712  -0.040728   0.056373  17.345544
 Mulliken charges:
               1
     1  C   -0.588913
     2  H    0.191772
     3  H    0.191780
     4  H    0.191760
     5  Br  -0.882944
     6  Cl  -0.103455
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.013601
     5  Br  -0.882944
     6  Cl  -0.103455
 Electronic spatial extent (au):  <R**2>=           1138.1694
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -1.3589    Y=              1.3612    Z=              2.4492  Tot=              3.1142
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -50.4684   YY=            -50.4850   ZZ=            -64.3150
   XY=              6.1710   XZ=             11.1046   YZ=            -11.1195
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              4.6211   YY=              4.6045   ZZ=             -9.2256
   XY=              6.1710   XZ=             11.1046   YZ=            -11.1195
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             14.7447  YYY=            -15.3293  ZZZ=            -37.0604  XYY=              7.1383
  XXY=             -6.0852  XXZ=            -12.4138  XZZ=             11.9723  YZZ=            -12.4866
  YYZ=            -11.5301  XYZ=              4.5399
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=           -331.6934 YYYY=           -333.1376 ZZZZ=           -950.7539 XXXY=            130.8510
 XXXZ=            236.8852 YYYX=            132.0741 YYYZ=           -236.6930 ZZZX=            259.1279
 ZZZY=           -260.3565 XXYY=           -114.5196 XXZZ=           -221.6323 YYZZ=           -221.0757
 XXYZ=            -85.6442 YYXZ=             85.8812 ZZXY=             60.4443
 N-N= 1.712278438309D+02 E-N=-7.695857882505D+03  KE= 3.070022237897D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE333\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -1.257329,-0.000067,0.000163\H,0,-0.974934,-0.884291,-0.550751\H,0,-0.
 97485,0.919148,-0.490094\H,0,-0.975005,-0.035097,1.0414\Br,0,1.648182,
 0.000011,-0.000018\Cl,0,-3.203884,0.000005,-0.000031\\Version=ES64L-G1
 6RevC.01\State=1-A\HF=-3074.3988584\RMSD=9.956e-10\Dipole=-1.2252148,-
 0.0000561,0.0001331\Quadrupole=-16.0349982,8.0175271,8.0174711,0.00005
 73,-0.0001373,-0.0000022\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 Time has a wonderful way of weeding out the trivial.
                                -- Richard Ben Sapir
 Job cpu time:       0 days  0 hours  0 minutes 29.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 15.1 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:52:08 2020.
